WanderLost/WanderLost/Client/Components/Card.razor

```diff
@@ -1,4 +1,4 @@
-﻿<CardFrame Title="@Title" Rarity="@TopRarity" ClassName="@ClassName">
+﻿<CardFrame Region="@Region" Zone="@Merchant.Zone" Rarity="@TopRarity" ClassName="@ClassName">
```

Comment:
property naming should be more detached, for this particular use case `Title` and `Subtitle` should do

Reply:
Good point, will change